openresty/Dockerfile-alpine

```diff
@@ -0,0 +1,253 @@
+# Current latest Openresty version is here.
+# Also specify the Nginx core used. (Openresty, adds their own version numbers after the nginx one, so we can't use the same number for both openresty and nginx)
+# Also specify the LUA version used. (Comes with the latest Openresty src bundle.)
+ARG OPENRESTY_VERSION="1.21.4.2" \
+    NGINX_VERSION="1.21.4" \
+    LUA_JIT_VERSION="2.1-20230410"
```

Comment:
Declare the ModSecurity version at the top so you can access it in all stages.

```suggestion
    LUA_JIT_VERSION="2.1-20230410"
ARG MODSEC_VERSION=3.0.9
```

Reply:
Should be OK now. Added MODSEC_VERSION at the start of the dockerfile and removed from other places.